src/shims/intrinsics/mod.rs

```diff
@@ -120,6 +120,18 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_bytes_ptr(ptr, iter::repeat(val_byte).take(byte_count.bytes_usize()))?;
             }
 
+            "ptr_mask" => {
+                let [ptr, mask] = check_arg_count(args)?;
+
+                let ptr = this.read_pointer(ptr)?;
+                #[allow(clippy::cast_sign_loss)]
+                let mask = this.read_scalar(mask)?.to_machine_isize(this)? as u64;
```

Comment:
This is equivalent to `to_machine_usize` I think? And `usize` is the type used in the intrinsic so I don't understand why the `isize` method should be called here.

Reply:
For some reason I couldn't find `to_machine_usize` when I was writing this, must be an overlook. Anyway, fixed now.